.github/workflows/compliance.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           run-tests: true
 
       - name: Send Slack notification if license check failed
-        uses: act10ns/slack@87c73aef9f8838eb6feae81589a6b1487a4a9e08
```

Comment:
This is considered a best practice to mitigate supply-chain attacks through github actions. I am happy to revisit the global policy in Haystack but for now I support going back to tags